sample/build.gradle.kts

```diff
@@ -25,13 +25,26 @@ android {
 }
 
 tasks.withType<KotlinCompile>().configureEach {
+  @Suppress("SuspiciousCollectionReassignment")
   kotlinOptions {
-    @Suppress("SuspiciousCollectionReassignment")
     freeCompilerArgs +=
       listOf(
         "-opt-in=androidx.compose.material3.ExperimentalMaterial3Api",
         "-opt-in=kotlinx.coroutines.ExperimentalCoroutinesApi"
       )
+
```

Comment:
From the article it saids:

>  You may find that you need to run the command above with --rerun-tasks, to ensure that the Compose Compiler runs, even when cached.

Is it worth adding a comment about that to call it out?

Reply:
I don't think that's accurate anymore. Compiler args are all inputs to the kotlinc task, and invalidate it if they change